crates/recursor/src/recursor.rs

```diff
@@ -587,24 +585,29 @@ mod for_dnssec {
     }
 }
 
-const RECOMMENDED_FILTER_RANGES: [&str; 17] = [
-    "10.0.0.0/8",      // RFC 1918 space
-    "172.16.0.0/12",   // RFC 1918 space
-    "192.168.0.0/16",  // RFC 1918 space
-    "100.64.0.0/10",   // CG NAT
-    "169.254.0.0/16",  // Link-local space
-    "192.0.0.0/24",    // IETF Reserved
-    "192.0.2.0/24",    // TEST-NET-1
-    "198.51.100.0/24", // TEST-NET-2
-    "203.0.113.0/24",  // TEST-NET-3
-    "240.0.0.0/4",     // Class E Reserved
-    "100::/64",        // v6 discard prefix
-    "2001:db8::/32",   // v6 documentation prefix
-    "3fff::/20",       // v6 documentation prefix
-    "5f00::/16",       // v6 segment routing prefix
-    "fc00::/7",        // v6 private address,
-    "fe80::/64",       // v6 link local
-    "ff00::/8",        // v6 multicast
+const RECOMMENDED_FILTER_RANGES: [&str; 22] = [
+    "127.0.0.0/8",        // Loopback range
```

Comment:
IMO it would be quite a bit nicer if we can use actual `IpNet` is here rather than storing strings here and convert it during construction. Looks like `IpNet::new_assert()` should be an option for this?